### ISSUE TITLE
rangeNotSelectedColor issue

### ISSUE THREAD

**Issue by TofPlay** [chicio/RangeUISlider#34]
**Describe the bug**
I try to change `rangeNotSelectedColor` 

**To Reproduce**
```swift
slideRange.rangeSelectedColor = .red
slideRange.rangeNotSelectedColor = .green
slideRange.leftKnobWidth = 20
slideRange.leftKnobHeight = 20
slideRange.leftKnobCorners = 10
slideRange.leftKnobColor = .red
slideRange.rightKnobWidth = 20
slideRange.rightKnobHeight = 20
slideRange.rightKnobCorners = 10
slideRange.rightKnobColor = .red
slideRange.delegate = self
```

**Expected behavior**
No select range with green color

**Screenshots**
![image](https://user-images.githubusercontent.com/1082222/107474269-5b788180-6b72-11eb-91b1-3bf8bb2b1f18.png)

**Device (please complete the following information):**
 - Model: iPhone Xr
 - OS: 14.4

**Additional context**
UIKit application

**Comment by chicio** [comment]
Hi @TofPlay ,

thank you for your feedback. Yes, you're right, with your indication I discovered a bug in the range note selected color/gradient setup. I will try to fix it today and release a new patch version.

Thank you.

PS remember to support my work by ⭐️ starring ⭐️ my repository

**Comment by chicio** [comment]
Hi @TofPlay ,

bug resolved and patch version 2.0.1 released on cocoa pods (and here on github). 
Thank you again.

Fabrizio.
